src/aiida/orm/nodes/data/structure.py

```diff
@@ -102,6 +102,15 @@ def has_pymatgen():
     return True
 
 
+def has_atomistic():
+    """:return: True if the StructureData and StructureDataMutable from aiida-atomistic module can be imported, False otherwise."""
+    try:
+        import aiida_atomistic  # noqa: F401
+    except ImportError:
+        return False
+    return True
```

Comment:
~~I'd suggest to move this try directly to where it is used.~~

Reply:
Okay, after seeing the test of dataclasses I see the use of this function, fine to have it here.